cdhweb/settings.py

```diff
@@ -423,15 +453,6 @@
     sys.modules[module_name] = module
     exec(open(f, "rb").read())
 
-# if in debug mode and django-debug-toolbar is available, add to installed apps
-if DEBUG:
-    try:
-        import debug_toolbar
-        INSTALLED_APPS.append('debug_toolbar')
-    except ImportError:
-        pass
-
-
```

Comment:
It's compatible and still worked for me. I moved this check to the urlconf instead of here because debug_toolbar is already in `OPTIONAL_APPS`, so it gets added automatically if available.

Reply:
I think `OPTIONAL_APPS` is a mezzanine thing! Pretty sure it isn't stock django, anyway; I can't find it anywhere in the settings documentation. (Although it's a nice solution...)

Reply:
Fixed the checks here and in the main urlconf.